test/src/test/java/hudson/cli/CLIActionTest.java

```diff
@@ -53,7 +51,7 @@
     public LoggerRule logging = new LoggerRule();
 
     @Test
-    @PresetData(DataSet.NO_ANONYMOUS_READACCESS)
+    @org.jvnet.hudson.test.recipes.PresetData(org.jvnet.hudson.test.recipes.PresetData.DataSet.NO_ANONYMOUS_READACCESS)
```

Comment:
What is this accomplishing? It is still deprecated.

Reply:
Instead of getting a bunch of warnings for importing the classes, one gets a single warning here.

Having 3 lines of warnings for a single offensive line is not better than having one warning for the same line.